week-2/Homework/extra/1-radio-stations.js

```diff
@@ -26,6 +37,13 @@
  */
 // `getStations` goes here
 
+function getStations() {
```

Comment:
Sorry for the confusion here. the helpful method name is isRadioStation not isRadioFrequcies. Would you like to give it another try